airflow/config_templates/config.yml

```
@@ -436,6 +322,124 @@
       example: ~
       default: ""
 
+- name: database
+  description: ~
+  options:
+    - name: sql_alchemy_conn
+      description: |
+        The SqlAlchemy connection string to the metadata database.
+        SqlAlchemy supports many different database engines.
+        More information here:
+        http://airflow.apache.org/docs/apache-airflow/stable/howto/set-up-database.html#database-uri
+      version_added: ~
```

```suggestion
      version_added: 2.2
```
In other options, we should set the same value.